src/ItspServices.pServer.Persistence.Sqlite/Repositories/SqlUserRepository.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Data.Common;
+
+[assembly: System.Runtime.CompilerServices.InternalsVisibleTo("ItspServices.pServer.ServerTest")]
+namespace ItspServices.pServer.Persistence.Sqlite.Repositories
+{
+    class SqlUserRepository
+    {
+        private DbProviderFactory _sqlFactory;
+        private readonly string _connectionString;
+
+        public SqlUserRepository(DbProviderFactory sqlFactory, SqlOptions options)
```

Comment:
I'm not 100% sure but I think you need to use `IOptions<SqlOptions> options` in order to make configured options work.